### PR TITLE
[Automotive] Add local deployment files to .gitignore

### PR DESCRIPTION
**Pull request by outofthecave**
## Description

These files are automatically generated when deploying the bot locally and therefore do not need to be tracked by git.

## Related Issue

I don't think we need an issue for this, but if you'd prefer me to create one, just let me know and I'll do so.

## Testing Steps

This does not apply here.

## Checklist
This does not apply here.

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the appropriate unit tests
- [ ] I have updated related documentation

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
- [ ] A duplicate issue is filed to track future  work.

If you have updated responses or `.lu` files:
- [ ] All languages have been updated
- [ ] You have tested deployment with your new models

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -339,4 +339,9 @@ ASALocalRun/
 /solutions/**/*.luis
 
 # QnAMaker json files
-/solutions/**/*.qna
\ No newline at end of file
+/solutions/**/*.qna
+
+# Automotive skill local deployment files
+/solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/.deployment
```
Comment (lauren-mills) [inline .gitignore]: This is a good call, but it'd be better to use a path with wildcards so all these files are ignored.

Comment (lzc850612) [reply .gitignore]: yes very good suggestion

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (outofthecave) [reply .gitignore]: Thanks!  I changed it to `/solutions/Virtual-Assistant/src/csharp/skills/*skill/*skill/.deployment` - I hope that's what you meant.  If not, let me know.

**Review by lauren-mills — APPROVED** [review]

**Comment by outofthecave** [comment]
@lzc850612 I made the changes you requested.  Can you have another look please?

**Comment by lzc850612** [comment]
I think it should be good enough to do 
/solutions/**/.deployment
/solutions/**/publish.cmd
/solutions/**/publish

They would be generated when we publish to azure and that's true for all web apps, not just skills.

Thanks!

**Pull request merged**